code/modules/food/nuggets.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/food/nugget
+	name           = "chicken nugget"
+	icon           = 'icons/obj/food/nuggets/nugget.dmi'
+	icon_state     = ICON_STATE_WORLD
+	nutriment_desc = "mild battered chicken"
+	nutriment_amt  = 6
+	nutriment_type = /decl/material/solid/organic/meat/chicken
+	material       = /decl/material/solid/organic/meat/chicken
+	bitesize       = 3
+	var/shape      = null
+	var/static/list/nugget_icons = list(
```

Comment:
will make it difficult for downstreams to add more modularly if they want (nuggets modpack when) but that's okay